### ISSUE TITLE
DX: Running the installing steps on a VM with php 7.0 errors silently

### ISSUE THREAD

**Issue by staabm** [deployphp/deployer#2101]
**Describe the bug**
run

```
curl -LO https://deployer.org/deployer.phar
php deployer.phar
```

and you get no errors at all and empty output.

expected result:

at least a error telling me, that my php version is not recent enough

**Comment by antonmedv** [comment]
Actually there is already check https://github.com/deployphp/deployer/blob/master/bin/dep#L17-L25

**Comment by staabm** [comment]
this check seem to exclude php 7.0 and the min requirement is 7.3 or similar

**Comment by antonmedv** [comment]
Will be cool to add it there.

**Comment by staabm** [comment]
fix submitted in #2102

**Comment by antonmedv** [comment]
I am cleaning up on issues. Fill free to reopen if needed. 
Also, we have a new section available: [discussions](https://github.com/deployphp/deployer/discussions).
